.github/workflows/ci.yml

```diff
@@ -2,14 +2,45 @@ name: CI-Build
 
 on:
   push:
-  pull_request:
   schedule:
     - cron:  '21 21 * * *'
   workflow_dispatch:
 
 jobs:
   build:
-    uses: axonivy-market/github-workflows/.github/workflows/ci.yml@v2
-    # Disable Web Tests
-    # with:
-    #   mvnArgs: -PnoWebTest
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Setup Java JDK
+      uses: actions/setup-java@v3
+      with:
+        java-version: 17
+        distribution: temurin
+
+    - name: Build with Maven
+      run: mvn clean verify --batch-mode ${{ inputs.mvnArgs }} ${{ secrets.mvnArgs }}
+
+    - name: Publish Unit Test Results
+      uses: EnricoMi/publish-unit-test-result-action@v2
+      if: always()
+      with:
+        junit_files: |
+          */target/*-reports/*.xml
+          !*/target/*-reports/failsafe-summary.xml
+    - name: Archive build artifact
+      uses: actions/upload-artifact@v3
+      with:
+        path: |
+          */target/*.iar
+          *product/target/*.zip
+    - name: Archive test reports
+      uses: actions/upload-artifact@v2
+      if: failure()
+      with:
+        name: test-reports
+        path: |
+          */target/testEngineOut.log
+          */target/selenide/*
+          */target/ivyEngine/deploy/*/*.deploymentLog
```

Comment:
hey @alexsuter 
after long and tedious random errors in this repo I've finally found a fix for these webtests ... and this archive helped a lot with it. 
so I'd like to have it in the standard pipeline, if possible by overwriting the existing v2 version, what do you think about it?
here's an example run, that actually failed > https://github.com/axonivy-market/persistence-utils/actions/runs/4819790603